packages/text/src/react/__specs__/code.spec.tsx

```diff
@@ -0,0 +1,25 @@
+import { convertStoriesToJestCases } from '@pluralsight/ps-design-system-util'
+import { render } from '@testing-library/react'
+import { axe } from 'jest-axe'
+import React from 'react'
+
+import * as stories from '../__stories__/code.story'
+
+describe('Text', () => {
+  describe('Code', () => {
+    const cases = convertStoriesToJestCases(stories)
+
+    it.todo('forwards the ref')
+
+    it.todo('supports polymorphism')
```

Comment:
why add the todos?

Reply:
because i feel like these components should support these features. i'm sure they will come up in the flexibility theme work that is planned